dask_ml/model_selection/_incremental.py

```diff
@@ -879,10 +879,17 @@ def _get_params(self):
     def _additional_calls(self, info):
         calls = {k: v[-1]["partial_fit_calls"] for k, v in info.items()}
 
+        if self.patience < 0 or self.patience == 1:
+            raise ValueError(
+                "patience={}<=1 will always detect a plateau. "
+                "this, set\n\n    patience >= 2"
+            )
         if self.patience and max(calls.values()) > 1:
             calls_so_far = {k: v[-1]["partial_fit_calls"] for k, v in info.items()}
             adapt_calls = {
-                k: [vi["partial_fit_calls"] + vi.get("_adapt", 0) for vi in v][-1]
+                k: [
```

Comment:
Can this change cause a IndexError? If none of the `vi in v` have an `_adapt` key, then we'll have a empty list, and take the last item of that, causing an IndexError.

Reply:
No. `_adapt` will always be in some element of the list because the function `_adapt` will be called first. Otherwise, every value in `calls` is 0 and this indexing will never be performed.